getClients.js

```diff
@@ -0,0 +1,22 @@
+exports.getAll = function(event, context, callback) {
+  const connection = db.connect(event.stageVariables);
+  var query = 'SELECT * FROM Clients ORDER BY' + event.queryStringParameters.sort';
+
+  connection.query(query, function (error, results, fields) {
+    if (error) {
+      db.close(connection);
+      callback(null, {
+        statusCode: 205,
```

Comment:
Status code 205 is incorrect here. 205 is used when server fulfills request and usually by RFC standards has zero length payload. A 500 status response with error details in body may be more relevant.